src/main/groovy/com/netgrif/application/engine/startup/ElasticsearchRunner.groovy

```diff
@@ -30,6 +31,9 @@ class ElasticsearchRunner extends AbstractOrderedCommandLineRunner {
     @Value('${spring.data.elasticsearch.index.task}')
     private String taskIndex
 
+    @Value('${spring.data.elasticsearch.index.uri}')
+    private String uriIndex
```

Comment:
feedback about property injection via a class was given in person. Commenting just to point out where to make the changes